VA-Explorer/va_explorer
# Add Logging to VA Explorer

PR to add logging to the system. This includes
- new logger configs in `config/settings/base.py` (one for click events and one for debugging info. Only writing to events.log)
- logging for key data management events
- logging for whenever user clicks run coding algorithm
- logging fo user login/logout
- logging for most dashboard events
- logging for downloading data
- logging for key data ingest tasks, such as date parsing and location matching

The logger exports session ids which facilitates tracking events by session. When a user first logs in, an entry is logged mapping their session id to username, so session IDs can be tied to the user. 

**How to Test**
- Log in
- Perform some of the actions listed below (for example, viewing a VA record, editing/saving/resetting, zoom in on dashboard, download data, etc.)
- Log out
- Your actions (including login and logout) should be logged in `/va_explorer/va_logs/events.log`

**TODO (in separate PR) logging for:** 
-   System 
         captured_error & error params
         get_total_warnings 
         get_total_errors 
- Dashboard
       download map
       download chart(s)
